.circleci/config.yml

```diff
@@ -426,9 +426,10 @@ jobs:
       - run:
           name: get wasmtime
           command: |
-            wget https://github.com/CraneStation/wasmtime/releases/download/dev/wasmtime-dev-x86_64-linux.tar.xz
-            tar -xf wasmtime-dev-x86_64-linux.tar.xz
-            cp wasmtime-dev-x86_64-linux/wasmtime ~/vms
+            # use a pinned version due to https://github.com/bytecodealliance/wasmtime/issues/714
```

Comment:
I think using a pinned copy is always makes for testing anyway,  So I wouldn't include this comment.. no explanation needed :)

Reply:
Well, at least the comment gives context for why it is as it is... but yeah, I get the argument for always pinning (even if I don't completely agree with it).